MàJ zoom en/fr

Aniort opened the pull request:
update zoom article fr/en
 Changes to be committed:
	modified:   src/en/articles/zoom/index.njk

modified:   src/fr/articles/zoom/index.njk

Commit:
Committer: ANIORT Vincent INNOV/IT-S <SLRQ4050@orange.com>
update zoom article fr/en
 Changes to be committed:
	modified:   src/en/articles/zoom/index.njk

modified:   src/fr/articles/zoom/index.njk

MewenLeHo commented on src/en/articles/zoom/index.njk:
```diff
@@ -2,7 +2,7 @@
 abstract: "Text size enlargement is used by many users (elderly people, people with visual impairments and people that should wear glasses and don’t…)"
 titleBeforeTag: true
-date: "2018-01-11"
+date: "2025-21-11"
```
```suggestion
date: "2018-01-11"
updateDate: "2025-11-21"
```

MewenLeHo reviewed (commented):
Fix dates, update dates and compilation

MewenLeHo committed:
Update src/fr/articles/zoom/index.njk

MewenLeHo committed:
Update src/en/articles/zoom/index.njk

MewenLeHo committed:
Merge branch 'master' into zoom

pya35 committed:
typo

pya35 commented:
@Aniort il manque la capture d'écran dans "../../images/zoom-firefox.png">

MewenLeHo commented:
@pya35: see https://github.com/Orange-OpenSource/a11y-guidelines/issues/772

Commit:
Committer: ANIORT Vincent INNOV/IT-S <SLRQ4050@orange.com>
image zoom firefox
 Changes to be committed:
	modified:   src/en/articles/zoom/index.njk
	modified:   src/fr/articles/zoom/index.njk

Commit:
Merge branch 'zoom' of https://github.com/Orange-OpenSource/a11y-guidelines into zoom

Commit:
Changes path to zoom Firefox images en/fr and to be committed:
	modified:   src/en/articles/zoom/index.njk
	modified:   src/fr/articles/zoom/index.njk

pya35 committed:
Merge branch 'master' into zoom

MewenLeHo committed:
Merge branch 'master' into zoom

MewenLeHo reviewed (changes requested):
Fix broken path for missing images.

MewenLeHo committed:
Update src/en/articles/zoom/index.njk

MewenLeHo committed:
Update src/fr/articles/zoom/index.njk

MewenLeHo committed:
Update src/en/articles/zoom/index.njk

MewenLeHo reviewed (approved):
OK

MewenLeHo committed:
Merge branch 'master' into zoom

MewenLeHo commented on src/en/articles/zoom/index.njk:
```diff
@@ -3,6 +3,7 @@ title: "Handling Zoom, enlarging text size"
 abstract: "Text size enlargement is used by many users (elderly people, people with visual impairments and people that should wear glasses and don’t…)"
 titleBeforeTag: true
 date: "2018-01-11"
+updateDate: "2025-11-21"
```
```suggestion
updateDate: "2025-11-28"
```

MewenLeHo commented on src/fr/articles/zoom/index.njk:
```diff
@@ -3,6 +3,7 @@ title: "Gestion de l’agrandissement des caractères"
 abstract: "Gestion du zoom, agrandissement de la taille du texte"
 titleBeforeTag: true
 date: "2018-01-11"
+updateDate: "2025-11-21"
```
```suggestion
updateDate: "2025-11-28"
```

MewenLeHo reviewed (approved):
Fix `updateDate` for release.

MewenLeHo committed:
Update src/en/articles/zoom/index.njk

MewenLeHo committed:
Update src/fr/articles/zoom/index.njk
